staging/src/k8s.io/apiserver/pkg/server/options/authentication.go

```diff
@@ -165,17 +185,145 @@ func (s *DelegatingAuthenticationOptions) ToAuthenticationConfig() (authenticato
 		return authenticatorfactory.DelegatingAuthenticatorConfig{}, err
 	}
 
+	clientCA, err := s.getClientCA()
```

Comment:
Are you ok with both `getClientCA` and `getRequestHeader` issuing a `GET` request to retrieve the same configmap?